pkg/helmexec/exec_test.go

```diff
@@ -675,17 +676,60 @@ exec: helm --kube-context dev fetch https://example_user:example_password@repo.e
 func Test_ChartPull(t *testing.T) {
 	var buffer bytes.Buffer
 	logger := NewLogger(&buffer, "debug")
-	helm := MockExecer(logger, "dev")
-	err := helm.ChartPull("chart", "--version", "1.2.3", "--untar", "--untardir", "/tmp/dir")
-	expected := `Pulling chart
+	tests := []struct {
+		name          string
+		helmBin       string
+		helmVersion   string
+		chartName     string
+		chartFlags    []string
+		listResult    string
+		expectedError string
+	}{
+		{
+			name:        "",
+			helmBin:     "helm",
+			helmVersion: "v3.6.0",
+			chartName:   "chart",
+			chartFlags:  []string{"--untar", "--untardir", "/tmp/dir"},
+			listResult: `Pulling chart
 Exporting chart
-exec: helm --kube-context dev chart pull chart --version 1.2.3 --untar --untardir /tmp/dir
-`
-	if err != nil {
-		t.Errorf("unexpected error: %v", err)
+exec: helm --kube-context dev chart pull chart --untar --untardir /tmp/dir
+`,
+			expectedError: "",
+		},
+		{
+			name:        "",
+			helmBin:     "helm",
+			helmVersion: "v3.9.0",
+			chartName:   "repo/helm-charts:0.14.0",
+			chartFlags:  []string{"--untardir", "/tmp/dir"},
+			listResult: `Pulling repo/helm-charts:0.14.0
+Exporting repo/helm-charts:0.14.0
+exec: helm --kube-context dev fetch oci://repo/helm-charts --version 0.14.0 --destination [\w/]+ --untardir /tmp/dir
+`,
+			expectedError: "",
```

Comment:
add case when expectedError not empty

Reply:
It is not used now, I removed this field